src/main/java/com/gravity9/mongocdc/MongoCDCManager.java

```diff
@@ -65,17 +63,17 @@ private Map<Integer, MongoChangeStreamWorker> createWorkers() {
      */
     public void start() {
         try {
-            logInContext(managerId, () -> log.info("Starting all workers for collection {}", clusterConfig.getCollection()));
+            log.info("{} - Starting all workers for collection {}", managerId, clusterConfig.getCollection());
             workers.values().forEach(worker -> {
                 runInNewThread(worker);
                 worker.awaitInitialization();
             });
-            logInContext(managerId, () -> log.info("All workers for collection {} are now ready!", clusterConfig.getCollection()));
+            log.info("{} - All workers for collection {} are now ready!", managerId, clusterConfig.getCollection());
         } catch (Exception ex) {
             try {
                 stop();
             } catch (Exception exception2) {
-                logInContext(managerId, () -> log.error("Stop on exception failed", exception2));
+                log.error(managerId + " - Stop on exception failed", exception2);
```

Comment:
you can also use here "{}" instead of " + ".

Reply:
done